### ISSUE TITLE
Give Windows users a batch file to run MyPaint with MYPAINT_DEBUG=1, with pause on exit

### ISSUE THREAD

**Issue by achadwick** [mypaint/mypaint#385]
We should provide Windows users with a way of running MyPaint in a CMD window so that they can see debug-level terminal output more easily. This will involve:

* Running from an installed batch file.
* Setting up the required environment variable (but, does this work in CMD? perhaps a command line argument would be better)
* Invoking "python2.exe" and not "python2w.exe" (so we'll have to aggregate that, or at least not delete it as we currently do when making the build)
  * In other words, this launcher will leave the CMD window open while MyPaint runs.
* Stopping after MyPaint finishes (PAUSE, I think.)

I'd like to put this sort of thing in a _Start → MyPaint w{64,32} → Test & Develop_ subfolder of the start menu if we can, along with the existing Github internet shortcut :)
